numpy/_core/fromnumeric.py

```diff
@@ -3588,24 +3588,42 @@ def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
 
     Notes
     -----
-    The standard deviation is the square root of the average of the squared
-    deviations from the mean, i.e., ``std = sqrt(mean(x))``, where
-    ``x = abs(a - a.mean())**2``.
-
-    The average squared deviation is typically calculated as ``x.sum() / N``,
-    where ``N = len(x)``. If, however, `ddof` is specified, the divisor
-    ``N - ddof`` is used instead. In standard statistical practice, ``ddof=1``
-    provides an unbiased estimator of the variance of the infinite population.
-    ``ddof=0`` provides a maximum likelihood estimate of the variance for
-    normally distributed variables. The standard deviation computed in this
-    function is the square root of the estimated variance, so even with
-    ``ddof=1``, it will not be an unbiased estimate of the standard deviation
-    per se.
+    The definition of the standard deviation of an array is not unique.
+    Assuming the input `a` is a one dimensional NumPy array and `mean` is
+    either provided as an argument or computed as ``a.mean()``, NumPy
+    computes the standard deviation of an array as::
+
+        N = len(a)
+        d2 = abs(a - mean)**2
```

Comment:
I guess `abs` is for the complex case?    If so - worth comment - as otherwise it is confusing.

Reply:
I guess. 
```suggestion
        d2 = abs(a - mean)**2  # abs for complex `a`
```